pkg/resource/aws/aws_s3_bucket_test.go

```diff
@@ -21,8 +21,9 @@ func TestAcc_AwsS3Bucket_BucketInUsEast1(t *testing.T) {
 					if err != nil {
 						t.Fatal(err)
 					}
-					result.AssertManagedCount(1)
+					result.AssertManagedCount(2)
```

Comment:
The filter is on type `aws_s3_bucket`, I count 3 bucket resources in your `s3.tf`. I don't understand why we assert on 2 here.

Reply:
Nice catch, I changed the tf file but forgot to run the test another time ...